cal_ratio_trainer/common/column_names.py

```diff
@@ -75,17 +75,35 @@ def _generate_col_mseg_names(index: int) -> List[str]:
 # list of cluster, track, and mseg names
 col_cluster_track_mseg_names = col_cluster_names + col_track_names + col_mseg_names
 
-
-col_jet_names = ["jet_pt", "jet_eta", "jet_phi"]
+col_jet_names = [
+    "jet_pt",
+    "jet_eta",
+    "jet_phi",
+    "jet_isClean_LooseBadLLP",
+    "jet_nn_calRatio_main_sig_highMass_v3Adv",
+    "jet_nn_calRatio_main_sig_lowMass_v3Adv",
+    "jet_nn_calRatio_main_bib_highMass_v3Adv",
+    "jet_nn_calRatio_main_bib_lowMass_v3Adv",
+    "jet_nn_calRatio_main_qcd_highMass_v3Adv",
+    "jet_nn_calRatio_main_qcd_lowMass_v3Adv",
+]
 
 
 col_llp_mass_names = ["llp_mH", "llp_mS"]
 
-event_level_names = [
-    "eventNumber",
-    "mcEventWeight",
-    "label",
-]
+event_level_names = ["label", "eventNumber", "mcEventWeight", "runNumber"]
+
+col_llp_names = ["llp_eta", "llp_phi", "llp_Lxy", "llp_Lz", "llp_pt"]
+
+print(
```

Comment:
This looks like a debugging statement. Use `logging.debug` or similar. Otherwise this gets printed out every run weather the user cares or not.